app/views/layouts/application.html.erb

```diff
@@ -2,6 +2,9 @@
 <html>
   <head>
```

Comment:
There is another error in the layout of the app. Kindly see the error below and make sure that the app runs without an error.
![err](https://user-images.githubusercontent.com/35504149/162589149-346bea7e-357a-4043-8f25-c747b2a3282e.png)